app/main/routes_frontend.py

```diff
@@ -676,3 +688,13 @@ def increment_zseries_recipe_id():
             recipe_id = r.id
 
     return recipe_id + 1
+
+def calcDisplayTemp(f_value):
+    if display_temperature() == 'C':
+        if f_value == 0:
+            return 0
+        else:
+            #Convert F to C
+            return round( (f_value - 32) / 1.8 )

```

Comment:
Make conversion a global function?

Reply:
this conversion function could easily be in a javascript source library purposed for temp conversion `temp_conversion.js` and all handling of conversions (min - max; base recipe changes; etc) can be handled there and leave everything else alone.